app/experimenter/experiments/models.py

```diff
@@ -196,6 +198,7 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         null=True,
     )
     takeaways_summary = models.TextField(blank=True, null=True)
+    _updated_date_time = models.DateTimeField(auto_now=True)
```

Comment:
Why does this one have an underscore at the beginning? Should it be `updated_date_time` to match the others?

Reply:
oh I see the other date fields are formatted like this too 🤔

Reply:
A single underscore is a Python convention for saying 'this is kinda private'

Reply:
its private 🔒

Reply:
Oh I thought I mentioned moving it down with the other fields.  Not a huge deal, just nice to keep things grouped together.

Reply:
@eliserichards 

https://peps.python.org/pep-0008/#descriptive-naming-styles

```
_single_leading_underscore: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.
```